Hi @felixfontein 

There was only one test failing and is not due to the test , but seems to be a bug in shippable, 

```
00:59 +Exception in thread QueueFeederThread (most likely raised during interpreter shutdown):
00:59 +Traceback (most recent call last):
00:59 +  File "/usr/lib64/python2.6/threading.py", line 532, in __bootstrap_inner
00:59 +  File "/usr/lib64/python2.6/threading.py", line 484, in run
00:59 +  File "/usr/lib64/python2.6/multiprocessing/queues.py", line 233, in _feed
00:59 +<type 'exceptions.TypeError'>: 'NoneType' object is not callable
```

--- a/changelogs/fragments/69578-shell-remote_tmp-quoting.yaml
+++ b/changelogs/fragments/69578-shell-remote_tmp-quoting.yaml
@@ -1,2 +1,2 @@
 bugfixes:
-  - shell - Fix quoting of mkdir command in creation of remote_tmp in order to allow spaces and other special characters (https://github.com/ansible/ansible/issues/69577).
+  - shell - fix quoting of mkdir command in creation of remote_tmp in order to allow spaces and other special characters (https://github.com/ansible/ansible/issues/69577).

